api/app/views/mno_enterprise/jpi/v1/admin/teams/_team.json.jbuilder

```diff
@@ -11,6 +11,6 @@ end
 json.app_instances do
   json.array! team.app_instances do |app_instance|
     json.extract! app_instance, :id, :name
-    json.logo app_instance&.app.logo&.to_s
+    json.logo app_instance.app.logo.to_s if app_instance.app
```

Comment:
Whats wrong with the safe navigation operator?

Reply:
Just think it's more explicit to do it this way